contrib/test/crio-integration-playbook.yaml

```diff
@@ -296,6 +296,10 @@
       regexp: 'overlay2\.override_kernel_check=1'
       state: present
     when: ansible_distribution == 'RedHat' or ansible_distribution == 'CentOS'
+  - name: Copy redhat sigstore file to test host
+    copy:
+      src: test/redhat_sigstore.yaml
```

Comment:
CI is failing because this path (relative to `contrib/test/` doesn't exist. I've created https://github.com/kubernetes-incubator/cri-o/pull/877 which carries this PR and fix this. Hopefully CI will be green over there :)

Reply:
The fix is this https://github.com/kubernetes-incubator/cri-o/pull/877/files#diff-8a06734a8d6b43be615cd22baa4ba999R301